huaweicloud/services/cbr/resource_huaweicloud_cbr_vault.go

```diff
@@ -586,17 +609,28 @@ func resourceVaultUpdate(ctx context.Context, d *schema.ResourceData, meta inter
 		opts.Name = d.Get("name").(string)
 	}
 
-	if d.HasChanges("size", "auto_expand") {
+	if d.HasChanges("size", "auto_expand", "auto_bind") {
 		if isPrePaid(d) {
 			return diag.Errorf("cannot update size or auto_expand if the vault is prepaid mode")
 		}
 		ae := d.Get("auto_expand").(bool)
+		ab := d.Get("auto_bind").(bool)
 		opts.AutoExpand = &ae
+		opts.AutoBind = &ab
 		opts.Billing = &vaults.BillingUpdate{
 			Size: d.Get("size").(int),
 		}
 	}
 
+	if d.HasChanges("bind_rules") {
```

Comment:
does the API works when only `bind_rules` changes?

Reply:
yes, it is.